src/client/content_cao.cpp

```diff
@@ -1622,7 +1622,7 @@ void GenericCAO::processMessage(const std::string &data)
 				m_reset_textures_timer = 0.05;
```

Comment:
the 2 comment lines above here should be removed too

Reply:
just removed it

Reply:
Still there

Reply:
oops, removed it in the other file but not there - now done